01_fibonacci/fib_test.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMainOutput(t *testing.T) {
+	// Given
+	r := require.New(t)
+	var buf bytes.Buffer
+	out = &buf
+
+	// When
+	main()
+
+	// Then
+	expected := strconv.Quote("0 1 1 2 3 5 8 13 21 34 \n")
```

Comment:
Not really necessary, but it would be nice to avoid the trailing space.

Reply:
Yes, removed trailing space

Reply:
Assignments asks to print the numbers on newline and also from 1 i.e.
1
1
2
3
5
8
13

Reply:
Done